notebooks/5-stellar-flares/interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>This is true, and good. Maybe we should emphasize the weakness of this model, too: namely, that it is slightly "trigger happy" in classifying something as a flare.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>